mix/CodeModel.cpp

```diff
@@ -55,9 +56,12 @@ CompilationResult::CompilationResult(const dev::solidity::CompilerStack& _compil
 {
 	if (!_compiler.getContractNames().empty())
 	{
-		m_contract.reset(new QContractDefinition(&_compiler.getContractDefinition(std::string())));
+		auto const& contractDefinition = _compiler.getContractDefinition(std::string());
+		m_contract.reset(new QContractDefinition(&contractDefinition));
 		m_bytes = _compiler.getBytecode();
 		m_assemblyCode = QString::fromStdString(dev::eth::disassemble(m_bytes));
+		dev::solidity::InterfaceHandler interfaceHandler;
+		m_contractDefinition = QString::fromStdString(*interfaceHandler.getABIInterface(contractDefinition));
```

Comment:
Why not call it `contractInterface`? In my interpretation, "definition" also includes e.g. private functions.